printer_mods/Badnoob/ABBN_for_Stock/Readme.md

```diff
@@ -0,0 +1,35 @@
+# AB-BN for Stock
+
+This is a revised Stock toolhead for the Dragon hotend. It takes the technology from AB-BN and ports it for use with the stock Afterburner Assembly.
+
+This allows you to use the stock Afterburner 4020 based part cooler and reduces the potential for jamming.
+
+The only real use case I know of for this is for someone who does not have a 5015 fan and is using a Dragon toolhead.
+
+#Why? 
```

Comment:
Please add an empty line after `#Why?`, otherwise it doesn't display as a heading.